data-structures/deque.md

```diff
@@ -34,8 +35,8 @@ Similar to a queue, deques exist as an ordered collection of items. A deque's na
 ### Materials
 
 - [Deque Geeks for Geeks](https://www.geeksforgeeks.org/deque-set-1-introduction-applications/)
-- [Deque Data Structure](https://youtu.be/5VDQxLAlfu0) - 2 minute video
-- [Stacks, Queues, & Double Ended Queues](https://youtu.be/IITnvmnfi_Y)
+- [Deque Data Structure](https://www.youtube.com/watch?v=kLBuJ1Hle8g) - 7 minute video
```

Comment:
The YouTube link that's was there earlier was also for the same thing. So would you please tell why you changed that ?

Reply:
Looks like the link has been changed and updated.